### PR TITLE
feat: extract artifacts from a unity build

### PR DESCRIPTION
**Pull request by eordano**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -3,7 +3,7 @@
   "version": "6.4.2",
   "license": "Apache-2.0",
   "scripts": {
-    "postinstall": "git lfs install --force",
+    "postinstall": "git lfs install --force && cp node_modules/decentraland-renderer/*.unityweb static/unity/Build/",
```
Comment (moliva) [inline package.json:6]:
```suggestion
    "postinstall": "git lfs install --force && cp -r node_modules/decentraland-renderer/Build static/unity/Build/",
```

Comment (moliva) [reply package.json:6]: Sorry, I understood we wanted everything from the build but now I realize that the JSON config and other files might be something that we want to update manually.

Comment (eordano) [reply package.json:6]: yes, specially vbecause CachedXMLRequest warnings keep coming up 😅

Comment (moliva) [reply package.json:6]: I would be concerned by the `unityVersion` mostly. Might only be a label, but we would be losing track of it potentially. 🤔

**Pull request merged**